tests/schema_evolution/code_gen/components_rename_member/evolution.yaml

```
@@ -1,8 +1,8 @@
-from_schema_version: 1
-to_schema_version: 2
-
-evolutions:
+migrations:
   nsp::TestComponent:
-    member_rename:
-      - y
-      - y_new
+    - from_version: 1
+      to_version: 2
+      change:
+        type: rename_member
```

```suggestion
      rename_member:
```
Make the type the key here instead of introducing an extra `change` sub-dict with a dedicated `type` key.